src/xo-app/vm/tab-disks.js

```diff
@@ -64,6 +65,95 @@ const parseBootOrder = bootOrder => {
   forEach(bootOptions, (text, id) => { order.push({id, text, active: false}) })
   return order
 }
+const COLUMNS = [
+  {
+    itemRenderer: (vdi, userData) =>
+      <Text value={vdi.name_label}
+        onChange={value => editVdi(vdi,
+                { name_label: value })}
+      />,
+    name: _('vdiNameLabel'),
+    default: true,
+    sortCriteria: 'name_label'
+  },
+  {
+    itemRenderer: (vdi, userData) =>
+      <Text value={vdi.name_description} onChange={value => editVdi(vdi, { name_description: value })} />,
+    name: _('vdiNameDescription'),
+    sortCriteria: 'name_description'
+  },
+  {
+    itemRenderer: (vdi, userData) =>
+      <Size value={vdi.size || null} onChange={size => editVdi(vdi, { size })} />,
+    name: _('vdiSize'),
+    sortCriteria: 'size'
+  },
+  {
+    itemRenderer: (vdi, userData) => {
+      const {srs, vm} = userData
+      const sr = srs[vdi.$SR]
+      if (sr === undefined) return null
+      return <XoSelect
+        onChange={sr => migrateVdi(vdi, sr)}
+        xoType='SR'
+        predicate={sr => sr.$pool === vm.$pool && isSrWritable(sr)}
+        labelProp='name_label'
+        value={sr}
+        useLongClick>
+        <Link to={`/srs/${sr.id}`}>{sr.name_label}</Link>
+      </XoSelect>
+    },
+    name: _('vdiSr'),
+    sortCriteria: sr => sr.name_label
+  },
+  {
+    itemRenderer: (vdi, userData) => {
+      const {vdis, vbds, vm} = userData
+      const vbd = find(vbds, vbd => vdis[vbd.VDI] === vdi)
+      if (vbd.is_cd_drive || !vdi) {
+        return null
+      }
+      if (vm.virtualizationMode === 'pv') {
+        return <Toggle
+          value={vbd.bootable}
+          onChange={bootable => setBootableVbd(vbd, bootable)}
+          />
+      }
+    },
+    name: _('vbdBootableStatus')
+  },
+  {
+    itemRenderer: (vdi, userData) => {
+      const {vdis, vbds, vm} = userData
+      const vbd = find(vbds, vbd => vdis[vbd.VDI] === vdi)
+      if (vbd.is_cd_drive || !vdi) {
+        return
+      }
+      return <StateButton
+        disabledLabel={_('vbdStatusDisconnected')}
+        disabledHandler={connectVbd}
+        disabledTooltip={_('vbdConnect')}
+        enabledLabel={_('vbdStatusConnected')}
+        enabledHandler={disconnectVbd}
+        enabledTooltip={_('vbdDisconnect')}
+
+        disabled={!(vbd.attached || isVmRunning(vm))}
+        handlerParam={vbd}
+        state={vbd.attached}
+          />
+    },
+    name: _('vbdStatus')
+  }
+]
+
+const COLUMNS2 = filter(COLUMNS, item => {
+  let {name} = item
+  let {props} = name
+  let {id} = props
```

Comment:
You don't need those intermediary variables + declare variables as `const` when you don't need to modify them later:
```js
const { id } = item.name.props
```